options/run.go

```diff
@@ -51,11 +51,11 @@ func (ro *RunOptions) AddFlags(cmd *cobra.Command) {
 	ro.ArchivistaOptions.AddFlags(cmd)
 	cmd.Flags().StringVarP(&ro.WorkingDir, "workingdir", "d", "", "Directory from which commands will run")
 	cmd.Flags().StringSliceVarP(&ro.Attestations, "attestations", "a", DefaultAttestors, "Attestations to record ('product' and 'material' are always recorded)")
-	cmd.Flags().StringSliceVar(&ro.Hashes, "hashes", []string{"sha256"}, "Hashes selected for digest calculation. Defaults to SHA256")
+	cmd.Flags().StringSliceVarP(&ro.Hashes, "hashes", "h", []string{"sha256"}, "Hashes selected for digest calculation. Defaults to SHA256")
 	cmd.Flags().StringVarP(&ro.OutFilePath, "outfile", "o", "", "File to write signed data to")
 	cmd.Flags().StringVarP(&ro.StepName, "step", "s", "", "Name of the step being run")
-	cmd.Flags().BoolVar(&ro.Tracing, "trace", false, "Enable tracing for the command")
-	cmd.Flags().StringSliceVar(&ro.TimestampServers, "timestamp-servers", []string{}, "Timestamp Authority Servers to use when signing envelope")
+	cmd.Flags().BoolVarP(&ro.Tracing, "trace", "t", false, "Enable tracing for the command")
+	cmd.Flags().StringSliceVarP(&ro.TimestampServers, "timestamp-servers", "u", []string{}, "Timestamp Authority Servers to use when signing envelope")
```

Comment:
I'm not a huge fan with `u` as the short-hand for TSA but, `t` is the only thing that makes sense for tracing... 😅. @ChaosInTheCRD or @mikhailswift either of you have a good suggestion?

Reply:
Completely agree @jkjell. I wasn't sure what else to use tbh, because as you noted `t` was already used aptly for tracing, as is `s` for StepName.

Maybe `a` for Authority? 

Let me know what y'all think. I can also understand if y'all believe some of these actually should not have single shorthand letter.

Reply:
Hey y'all 👋 . Sorry to bother y'all, but any update on this?

Reply:
For some reason, there's an issue with Cobra for these changes... 🤔 

> panic: unable to redefine 'h' shorthand in "run" flagset: it's already used for "hashes" flag

☝️ is in the e2e test and docs GHA workflows for some reason.

Reply:
hmm let me take a look, sorry

Reply:
ahh yea I think the issue is can't overwrite help shorthand -h. Doh

Reply:
with regards to timestamp servers and tracing, may be better with `r` and `t` respectively, but that's just my personal preference. 

I'm starting to think, this PR can get trashed.

Reply:
hey @jkjell, did you see my previous comment. Do you think I should close this PR, or do you think it would still be beneficial to have some of the run options have a shorthand variable? I think we can scrap trying to force all run options to have a shorthand variable, but I think it would be beneficial to have hashes have a shorthand variable. What do you think? I'm also happy to close this PR and open an issue to discuss there if needed.

Reply:
Let's go with `r` and `t`. 👍 I thought about recommending `d` for hashes but, then I figured we'd get a lot of grief for not understand the difference between hashing algorithms vs their resulting digest. 😂

Reply:
hey @jkjell FYI just pushed my updates.